cmd/bridge/main.go

```diff
@@ -266,10 +267,28 @@ func main() {
 	case "off-cluster":
 		k8sEndpoint = validateFlagIsURL("k8s-mode-off-cluster-endpoint", *fK8sModeOffClusterEndpoint)
 
+		tlsConfig := &tls.Config{}
+		if *fK8sModeOffClusterSkipVerifyTLS {
+			tlsConfig.InsecureSkipVerify = *fK8sModeOffClusterSkipVerifyTLS
+		} else {
+			k8sOffClusterCA := ""
+			if *fK8sModeOffClusterCAFile != "" {
+				k8sOffClusterCA = *fK8sModeOffClusterCAFile
+			} else {
+				k8sOffClusterCA = *fCAFile
+			}
+			rootCAs := x509.NewCertPool()
+			ca, err := ioutil.ReadFile(k8sOffClusterCA)
+			if err != nil {
+				log.Fatalf("Read k8sOffClusterCA fail")
```

Comment:
It's not a good practice to use variable names in `log`, also the `err` needs to be attached to the output
```
log.Fatalf("Failed to read k8s-mode-off-cluster CA file: %v", err)
```

Reply:
Thanks for your suggestion